Sources/Misc/PriceFormatterProvider.swift

```diff
@@ -15,27 +15,61 @@ import Foundation
 
 /// A `NumberFormatter` provider class for prices.
 /// This provider caches the formatter to improve the performance.
-final class PriceFormatterProvider: Sendable {
+public final class PriceFormatterProvider: Sendable {
```

Comment:
I wonder, why should this API be public? Its `init` is not public.
It seems to me like it's only public to allow `StoreProduct`'s new `init` be public
```swift
convenience init(
    sk2Product: SK2Product,
    priceFormatterProvider: PriceFormatterProvider
)
```
But I think that one shouldn't be public either?